src/mono/wasm/debugger/BrowserDebugProxy/DebugStore.cs

```diff
@@ -760,6 +769,7 @@ public IEnumerable<SourceFile> Add(SessionId sessionId, byte[] assembly_data, by
             }
             catch (Exception e)
             {
+                System.Console.WriteLine($"Failed to load assembly: ({e.Message})");
```

Comment:
Sorry, you can drop this. I missed the error because the proxy log level was set to information. my mistake!

```
dbug: Microsoft.WebAssembly.Diagnostics.DevToolsProxy[0]
      Failed to load assembly: (Failed to resolve assembly: 'library-dependency-debugger-test2, Version=6.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35')
```

```suggestion
```